VERSION

```diff
@@ -1 +1 @@
-1.7.1-dev
+2.0.0
```

Comment:
It looks like a newline was removed from this file

Reply:
Fixed